cohort-engine/src/main/java/com/ibm/cohort/engine/measure/CDMMeasureEvaluation.java

```diff
@@ -27,7 +37,16 @@
 
 	public static final String CARE_GAP = "care-gap";
 	public static final String CDM_CODE_SYSTEM_MEASURE_POPULATION_TYPE = "http://ibm.com/fhir/cdm/CodeSystem/measure-population-type";
-
+	
+	public static final String EVIDENCE = "measure-report-evidence";
+	public static final String EVIDENCE_URL = "http://ibm.com/fhir/cdm/StructureDefinition/measure-report-evidence";
+	
+	public static final String EVIDENCE_TEXT = "measure-report-evidence-text";
+	public static final String EVIDENCE_TEXT_URL = "http://ibm.com/fhir/cdm/StructureDefinition/measure-report-evidence-text";
+	
+	public static final String EVIDENCE_VALUE = "measure-report-evidence-value";
+	public static final String EVIDENCE_VALUE_URL = "http://ibm.com/fhir/cdm/StructureDefinition/measure-report-evidence-value";
```

Comment:
Agree that a CDM constants class would be nice, will put together another PR for that change to keep this change set cleaner.

Reply:
Went the easy route and just added the constants to this PR 🤷 Didn't move Dan's as it's not CDM right now, but assume it will be and should be moved after this morning's conversation.